modules/_node-scripts/index.mjs

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
```

Comment:
I dont think this is needed since we specify that in the `bin` file

Reply:
done